build.sbt

```diff
@@ -5,151 +5,154 @@ import pl.project13.scala.sbt._
 lazy val buildSettings = inThisBuild(
   Seq(
     organization := "io.aecor",
-    scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12", "2.12.4")
+    scalaVersion := "2.12.4"
   )
 )
 
-lazy val akkaVersion = "2.5.9"
-lazy val akkaPersistenceCassandraVersion = "0.59"
-lazy val akkaPersistenceJdbcVersion = "3.2.0"
-lazy val catsVersion = "1.1.0"
-lazy val catsEffectVersion = "0.10"
+lazy val akkaVersion = "2.5.15"
+lazy val akkaPersistenceCassandraVersion = "0.61"
+
+lazy val catsVersion = "1.4.0"
+lazy val catsEffectVersion = "1.0.0"
+lazy val scodecVersion = "1.10.4"
 lazy val logbackVersion = "1.1.7"
 lazy val cassandraDriverExtrasVersion = "3.1.0"
 lazy val jsr305Version = "3.0.1"
 lazy val boopickleVersion = "1.3.0"
 lazy val monocleVersion = "1.5.0-cats"
+lazy val fs2Version = "1.0.0"
+lazy val log4catsVersion = "0.2.0-M1"
 
-lazy val monixVersion = "3.0.0-M3"
 lazy val scalaCheckVersion = "1.13.4"
-lazy val scalaTestVersion = "3.0.1"
+lazy val scalaTestVersion = "3.0.5"
 lazy val scalaCheckShapelessVersion = "1.1.4"
 lazy val shapelessVersion = "2.3.3"
-lazy val kindProjectorVersion = "0.9.4"
+lazy val kindProjectorVersion = "0.9.7"
 lazy val scalametaVersion = "1.8.0"
 
 // Example dependencies
 
-lazy val circeVersion = "0.9.0"
-lazy val akkaHttpVersion = "10.0.11"
-lazy val akkaHttpJsonVersion = "1.19.0"
+lazy val circeVersion = "0.9.3"
+lazy val http4sVersion = "0.20.0-M4"
```

Comment:
:thinking: the latest I can find is 0.20.0-M1

Reply:
Right, I might have looked at `v0.19.0-M4`, too much coffee yesterday.